README.md

```diff
@@ -56,6 +56,7 @@ Many models of Orange Pi hardware running [Armbian](https://www.armbian.com/) ar
 | `WITH_H_NTP`                    | None                                           | Set to `true` if you want to use a Hyperboria-compatible NTP server: `fc4d:c8e5:9efe:9ac2:8e72:fcf7:6ce8:39dc` |
 | `WITH_FAKE_HWCLOCK`             | None                                           | Set to `true` if you want to force hwclock to store its time every 5 minutes. |
 | `WITH_EXTRA_TOOLS`              | None                                           | Set to `true` if you want to install non-essential tools useful for network analysis: vim socat oping bmon iperf3 |
+| `WITH_MESH_ID`                  | None                                           | Set the mesh ID for use with `WITH_MESH_POINT` |
```

Comment:
These are all boolean flags to enable/disable features. I don't want to start piling other configs to this list. Can we get the mesh ID some other way? How do we get the WPA2 password?